src/main/java/com/fcgl/madrid/config/ElasticConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.fcgl.madrid.config;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.transport.client.PreBuiltTransportClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.transport.TransportAddress;
+
+/**
+ * Configuration for Elastic Search
+ * TODO: move static text to a properties file
+ */
+@Configuration
+public class ElasticConfiguration{
+
```

Comment:
Modifying this as stated in the instructions doesn't work. Spring complains about invalid IPv6 address.